### DISCUSSION TITLE
pH daily limit

### DISCUSSION THREAD

**Discussion by katodude** [tagyoureit/nodejs-poolController#889] [Nixie] [answered]
I am getting a pH daily limit error in the Nixie. What do I need to turn this off?

![IMG_0265](https://github.com/tagyoureit/nodejs-poolController/assets/66255504/9979e8ca-0c9d-4e4a-9874-64b44ffe8c32)

**Comment by truvec** [comment, marked as answer]
In settings  Max limit per rolling.
<img width="390" alt="Screenshot 2024-01-23 at 8 14 41 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/239790/df171718-ec11-4e0b-9f78-a6a0a91c4bf6">

**Reply by katodude** [reply]
Well that fixed it. 

Weird if I have dose set to manual the Max Limit options do not appear. Changed it up and it now works.

**Reply by tagyoureit** [reply]
That alert shouldn't show if the dosing method is set to manual.  If you want to go shock your pool that's up to you.  I put a check in the code to not display the error.  (It shouldn't have actually prevented you from dosing, but it would be annoying.)
